server/graphql/types/src/types/name.rs

```diff
@@ -239,7 +243,9 @@ mod test {
                             r.address1 = Some("address1".to_string());
                             r.address2 = Some("address2".to_string());
                             r.created_datetime =
-                                Some(NaiveDate::from_ymd(2022, 05, 18).and_hms(12, 07, 12))
+                                Some(NaiveDate::from_ymd(2022, 05, 18).and_hms(12, 07, 12));
+                            r.date_of_birth =
+                                Some(NaiveDate::from_ymd(1995, 05, 15).and_hms(00, 00, 00))
```

Comment:
date_of_birth is a NaiveDate, adding `and_hms()` makes it a NaiveDateTime.

I would also add a trailing `;` A bit confusing why Rust allows to have none here. Here  is my explanation: Not having a trailing `;` means the result of the statement or expression is returned. The assignment  `r.date_of_birth = ... ` has the type `()` ("void") which is a valid return value here. While this is valid its IMHO not what you would expect...